src/util.js

```diff
@@ -156,7 +156,7 @@ class Logger {
             this.debug("server", this.hideLog);
         }
     }
-    log(module, msg, level) {
```

Comment:
`msg` and `level` have to be swapped.

This part was refactored by JetBrains IDE, all usages were updated automatically too.